CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * [CHANGE] Ingester: `/ingester/flush` endpoint is now only allowed to execute only while the ingester is in `Running` state. The 503 status code is returned if the endpoint is called while the ingester is not in `Running` state. #7486
 * [FEATURE] Store-gateway: Allow specific tenants to be enabled or disabled via `-store-gateway.enabled-tenants` or `-store-gateway.disabled-tenants` CLI flags or their corresponding YAML settings. #7653
 * [FEATURE] New `-<prefix>.s3.bucket-lookup-type` flag configures lookup style type, used to access bucket in s3 compatible providers. #7684
+* [FEATURE] Server: added experimental HTTP PROXY protocol support. The PROXY protocol support can be enabled via `-server.proxy-protocol-enabled=true`. #7698
```

Comment:
Does Mimir now act as HTTP proxy? I don't understand this feature.

Reply:
Oh, it's referring to this: https://www.haproxy.org/download/2.3/doc/proxy-protocol.txt

Reply:
It's a way to communicate information about client at the beginning of TCP connection. It's unrelated to HTTP, but this flag enables it for HTTP server. Is it also enabled for gRPC server?

Reply:
> Is it also enabled for gRPC server?

Yes, it is. If `-server.proxy-protocol-enabled=true` then both HTTP and gRPC servers will expect to read [PROXY protocol header](https://www.haproxy.org/download/2.3/doc/proxy-protocol.txt) before accepting connection.